utils.go

```diff
@@ -360,3 +360,55 @@ func skipFloat(r byteReader) error {
 
 	return nil
 }
+
+func checkRdbModuleOpCode(r byteReader, expected int) error {
+	val, err := readLength(r)
+	if err != nil {
+		return err
+	}
+
+	if val != expected {
+		return fmt.Errorf("illegal rdbModuleOpcode %d, expect:%d", val, expected)
```

Comment:
I think it would be better to create an error type in this case.

Reply:
fixed https://github.com/tommy351/rdb-go/pull/4/commits/59fe907f69376ba181e59a6567f7ea88e70b9313